.github/workflows/tests.yml

```diff
@@ -0,0 +1,61 @@
+on:
+  workflow_call:
+name: Tests
+jobs:
+  run:
+    strategy:
+      matrix:
+        go-version: [1.16.x, 1.17.x]
+        os: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Setup - Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
+
+      - name: Setup - Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Install Python dependencies
+        run: |
+          python -m pip install --upgrade iso8601 requests securesystemslib six tuf
```

Comment:
Should we be pinning the python dependencies, so that a release of one of these python packages doesn't cause a change in CI behaviour? (counter-argument – if we pin them, we need to monitor and update them).

Reply:
Addressed it via a requirements file which is also monitored by dependabot 👍